gravitee-apim-console-webui/src/management/api/general-info/api-general-info.component.ts

```diff
@@ -102,10 +102,12 @@ export class ApiGeneralInfoComponent implements OnInit, OnDestroy {
 
     this.isQualityEnabled = this.constants.env?.settings?.apiQualityMetrics?.enabled;
 
-    this.apiId = this.activatedRoute.snapshot.params.apiId;
-
-    combineLatest([this.apiService.get(this.apiId), this.categoryService.list()])
+    this.activatedRoute.params
```

Comment:
I'm suprised that we need to reload the component like this when we have a this.router.navigate(['../', apiDuplicated.id], { relativeTo: this.activatedRoute })), when we close the duplicate dialog. Have you checked why this is not working ?

Reply:
Yes. The this.router.navigate is changing the URL and then if you visit any page, it will be with the new api Id but for info page, it's the same component from where the URL is getting changed. In the URL, only parameters are getting changed, so ngOnInit will not be called again and hence no backend call.